### PR TITLE
Cache property specific keyframes

### PR DESCRIPTION
**Pull request by steveblock**

### CODE REVIEW

**Comment by steveblock** [comment]
Not yet fully tested - waiting for Travis

File: web-animations.js [hunk 1]
```diff
@@ -1794,24 +1794,20 @@ KeyframeAnimationEffect.prototype = createObject(AnimationEffect.prototype, {
...
       exitModifyCurrentAnimationState(true);
     }
   },
   _sample: function(timeFraction, currentIteration, target) {
-    var properties = this._getProperties();
-    for (var i = 0; i < properties.length; i++) {
-      compositor.setAnimatedValue(target, properties[i],
-          this._sampleForProperty(timeFraction, currentIteration,
-              properties[i]));
+    var frames = this._propertySpecificKeyframes(property);
```
Comment (dstoc) [inline web-animations.js]: property is not needed here

Comment (steveblock) [inline web-animations.js]: Fixed

**Comment by steveblock** [comment]
Travis build passed - ready for a look

**Pull request merged**